### ISSUE TITLE
Network training guidance and model

### ISSUE THREAD

**Issue by zhanghanduo** [rpng/calc2.0#4]
Hi. Thanks for the inspirational work!

Just want to ask could you directly provide a downloadable link of COCO stuff weight file? If you have time, could you also share some of your tips of training this network like learning rate and stopping conditions because they are not explained in detail in the paper.  Also if I want the function of "resuming latest epoch of model", I have to modify the code right?
Thanks again.

**Comment by nmerrill67** [comment]
Hi thanks for the interest. Did you mean the network weights or the loss weights? If you mean the loss weights, they are available in `dataset/loss_weights.txt`. I can provide network weights as well if that's what you meant. 

As for training, I am using the parameters listed in the code (default Adam parameters). I stopped the model when is did not get any better testing on the CampusLoop dataset. This test is automatically run periodically during training as the eval functionality, and can be viewed from tensorboard under AUC.

**Comment by chuan573906361** [comment]
Hi Nathaniel, Thanks for the remarkable work!
I've trained the net on the coco-2017 dataset.then I tested the model on the campusLoopDataset provided in the repo.but it  seems the performance is too much lower than the results in the paper.so could you please send me the model? jinchuan1zhang@gmail.com

**Comment by wtiandong** [comment]
Hi @nmerrill67 ,
Thanks for your great work! Could you send me the network weights to wtiandong@gmail.com, please?

**Comment by dianqiliuze** [comment]
Hi Nathaniel, Thanks for your work! Could you please send me the network weights to 3011296707@qq.com? thank you very much!

**Comment by garriton** [comment]
Hi Nathaniel, Thanks for your work! Could you please send me the network weights to zhgyue@gmail.com? thank you very much!

**Comment by chenjiangtao229** [comment]
Hi Nathaniel, Thanks for your work! Could you please send me the network weights to chenjiangtao33@163.com? Thank you very much!

**Comment by zhuhu00** [comment]
Hi @nmerrill67 ,
Thanks for your great work! Could you send me the network weights to huzhu00@gmail.com, please? Thank you very much.

**Comment by jialeB** [comment]
Hi, @nmerrill67 ，
Thanks for the inspirational work! Could you send me the network weights to bai_jiale123@163.com, please? Thank you very much.

**Comment by xiaozhang404-coding** [comment]
I also trained the network at coco-2017, but after I tested the model, I found that the performance is lower than the results of the paper, can you send your model? My mailbox is zhangbaibai404@gmail.com

**Comment by dianqiliuze** [comment]
您好，您的邮件我已收到，谢谢！

**Comment by xiaozhang404-coding** [comment]
Hi Nathaniel, Thanks for your work!I also trained the network at coco-2017, but after I tested the model, I found that the performance is lower than the results of the paper, can you send your model? My mailbox is zhangbaibai404@gmail.com

**Comment by zhanghao328** [comment]
Hi Nathaniel, Thanks for your work! I want to use it in my own slam system but my GPU is so old that I can not train the net.  So can you send your model and I would appreciate that. My mailbox is 1077883001@qq.com.

**Comment by dianqiliuze** [comment]
您好，您的邮件我已收到，谢谢！

**Comment by dingshuoshuo** [comment]
Hi, @nmerrill67 ，
Thanks for the inspirational work! Could you send me the network weights to 43205504@qq.com, please? Thank you very much.

**Comment by dianqiliuze** [comment]
您好，您的邮件我已收到，谢谢！

**Comment by linqi-wang** [comment]
Hi, @nmerrill67 ，
Thanks for the inspirational work! I'm a beginner studying loop closure and have been working on reproducing the results of this paper for nearly a month. However, I keep encountering issues in various parts of the implementation, so I would greatly appreciate your assistance. Could you send the network weights and loss weights to my email at 1878817809@qq.com? Thank you very much again!
